dts/bindings/sram/intel,adsp-settings.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel ADSP SRAM settings.
+
+compatible: "intel,sram-settings"
```

Comment:
Would these not be properties of some SRAM node?

Reply:
That makes some sense, at least for the hp-sram and l2 bits, which have nodes. Not sure yet about the L1 side - will look into that.

Reply:
Agree with @galak, it looks to me like a device properties

Reply:
For the "init HP SRAM" that was fairly easy, so done on v2. But I kept the others ("disable l2 cache at boot" and "power on SRAM cache") in this "global" way as there's no single node associated with what these config. Further thoughts?